components/WalletStatus.vue

```diff
@@ -31,19 +54,32 @@ export default {
             }));
         },
     },
+    watch: {
+        error(nv){
+            this.errorDialog = nv !== ''
+        }
+    },
     mounted() {
         this.showInstallMetaMask = !window.ethereum;
     },
     methods: {
         ...mapActions('wallet', ['initWallet', 'switchNetwork']),
         async onChangeNetwork(e) {
             this.networkValue = e;
-            await this.switchNetwork(e);
+            try {
+                await this.switchNetwork(e);
+                this.error = ''
+            } catch (error) {
+                // eslint-disable-next-line no-console
+                console.error(error);
+                this.error = error;
+            }
         },
         async onClickConnectWallet() {
             let res = false;
             try {
                 res = await this.initWallet();
+                this.error = ''
```

Comment:
we should reset `this.error` before `await`?
```suggestion
                this.error = ''
                res = await this.initWallet();
```